mkdocs.yml

```diff
@@ -35,19 +40,32 @@ markdown_extensions:
   # https://pypi.org/project/mdx-truly-sane-lists/
   - mdx_truly_sane_lists
 
+  # Embed external content
+  - pymdownx.snippets
+
   # allows indented code blocks inside lists
-  - pymdownx.superfences
+  - pymdownx.superfences:
       custom_fences:
         - name: mermaid
           class: mermaid
           format: !!python/name:pymdownx.superfences.fence_code_format
+
+  - pymdownx.details
```

Comment:
allows collapsible info blocks

Reply:
As above, maybe this kind of note belongs in a comment in the source.